kratos/geometries/triangle_2d_3.h

```diff
@@ -436,20 +436,20 @@ template<class TPointType> class Triangle2D3
      * http://fileadmin.cs.lth.se/cs/personal/tomas_akenine-moller/code/tribox_tam.pdf
      * 
      * @return bool if the triangle overlaps a box
-     * @param rLowPoint first corner of the box
-     * @param rHighPoint second corner of the box
+     * @param r_low_point first corner of the box
+     * @param r_high_point second corner of the box
      */
-    bool HasIntersection( const Point<3, double>& rLowPoint, const Point<3, double>& rHighPoint ) override 
+    bool HasIntersection( const Point<3, double>& r_low_point, const Point<3, double>& r_high_point ) override 
```

Comment:
@miguelmaso function arguments must be camelcase (so no change is needed in here).

Reply:
👍 for @rubenzorrilla